%YAML:1.0
---
caliberationTimeChar: "Thu Nov  9 21:50:40 2023\n"
caliberationTime_t: 1699537840
imageWidth: 1280
imageHeight: 1024
cameraMatrix: !!opencv-matrix
   rows: 3
   cols: 3
   dt: d
   data: [ 8.8136100819112039e+02, 0., 6.4220272461712216e+02, 0.,
       8.8058628963515389e+02, 5.0584011804970760e+02, 0., 0., 1. ]
disCoffes: !!opencv-matrix
   rows: 1
   cols: 5
   dt: d
   data: [ -9.3578985363918982e-02, 7.3018505781808796e-02,
       -3.8644741900791793e-04, -9.8636591357284249e-05,
       1.4049603863372971e-02 ]
rvecs: !!opencv-matrix
   rows: 33
   cols: 1
   dt: "3d"
   data: [ -9.4054031497869492e-02, -5.9147558222699473e-02,
       -1.5373971918603226e+00, -4.9839896942115262e-02,
       -1.3411297613424689e-01, -2.4850750250767906e+00,
       1.0582701614730378e-01, 1.8579870743107774e-01,
       2.4009141685653770e+00, -6.6272030696363504e-02,
       7.0345552542341905e-01, -3.0505365437058329e+00,
       2.8419808372549329e-03, -3.4290064650084001e-01,
       3.0511527730798553e+00, 1.7015567966806178e-01,
       1.5890070266573147e-01, -3.0518026392595723e+00,
       8.4701165663012612e-01, 4.7381595527533216e-01,
       2.9345988319367371e+00, -1.0785342775678344e+00,
       3.8540402779877969e-01, 2.9139365011894540e+00,
       7.1342033510931369e-01, 3.8956255973750303e-01,
       -1.4441748036532589e+00, -4.1704495103972788e-01,
       -4.1279380279856831e-01, -1.4904071311350118e+00,
       -5.4576363885538770e-01, -1.9009845740990908e-02,
       -8.8910810548346697e-01, 4.9802409420097525e-02,
       -1.3547013858181800e-02, -1.6126467944214340e+00,
       7.8752509573253759e-01, 1.7384566435220589e-01,
       -2.0045805409782318e+00, -1.8976786643700211e-01,
       -7.4981423815568482e-01, 2.2628787830378192e+00,
       7.7700343948553829e-01, 9.1324487240475524e-02,
       -4.1773735232088806e-02, -2.6241238433126451e-01,
       -3.7135957255354468e-01, 1.6665016137286452e+00,
       7.5598794871982455e-01, -8.1939386323413377e-01,
       1.4554494935686444e+00, 7.8087760954484153e-01,
       3.2699594607433086e-01, 1.6135482080522307e-02,
       4.4055716126993311e-01, 8.4875281187964469e-01,
       1.4078305471989490e+00, 5.5574466402616540e-01,
       7.4158808776192064e-01, 5.7223826698734082e-01,
       4.9951451048401110e-02, -1.0716177872567541e-02,
       -1.6162927120275223e+00, 8.4309460957968052e-01,
       5.6181684599300941e-01, 2.0168501406990544e+00,
       6.1044071710901593e-01, 1.1147474017052292e-01,
       -6.0583328145488058e-01, 8.6075245180276516e-01,
       -5.4616094487334077e-01, 9.5139439234206667e-03,
       -9.9667964442934054e-01, 5.4396388424854947e-01,
       1.6900771597316098e+00, 1.8528130484820318e-01,
       -5.8624825426419980e-01, 8.2133683212392861e-01,
       3.3326236360798772e-01, -7.3186261102955530e-01,
       -6.2180588181792673e-01, 2.4554315169090637e-02,
       -1.7331207079427967e-03, -1.5801823946477171e+00,
       -6.8920776623884530e-02, 1.0359898535778310e-01,
       -1.6014905026761987e+00, -7.5426946177551843e-02,
       3.1701106447813591e-02, -1.5949019593251370e+00,
       -1.3694619354105034e-01, -3.0346194228260018e-02,
       -3.1060833464200539e+00, -3.6619879104235385e-02,
       -4.9027774038358701e-02, -3.1317522026240305e+00,
       6.3653005467145501e-02, -1.6379335442937079e-01,
       -3.0983697325231732e+00 ]
tvecs: !!opencv-matrix
   rows: 33
   cols: 1
   dt: "3d"
   data: [ -1.1701237101826231e+02, 6.5287674942759267e+01,
       2.2526565097665500e+02, -2.3283591150956518e+01,
       1.2297644939714850e+02, 3.1048526078464340e+02,
       1.4664676335301991e+02, 2.4702510090534453e+01,
       2.8858418197848954e+02, 5.5046977748589825e+01,
       1.4896249295480592e+02, 3.2026270031847906e+02,
       1.5444828699461411e+02, 1.3135863974761017e+02,
       2.9630298109428259e+02, -3.8916462422535936e+01,
       1.3835741826773636e+02, 3.2479774816131334e+02,
       7.3793610330112159e+01, 8.8839175146360759e+01,
       2.2286671780551637e+02, 6.4433515218760192e+01,
       1.0847022406209226e+02, 3.3610812662433148e+02,
       -1.1179901942936699e+02, 6.7802003763956805e+01,
       3.0392062074698350e+02, -1.1476842533177849e+02,
       4.4648265859872978e+01, 2.4613087157936027e+02,
       -2.2750427856904270e+02, -4.8353164427297898e+01,
       4.1278160252904496e+02, -1.3849724545784858e+02,
       3.8545199219820930e+01, 2.4575383607289950e+02,
       -5.7934816233496250e+01, 1.3851612608102113e+02,
       3.5044059176887566e+02, 1.8139593864312315e+02,
       5.1923196083365646e+01, 3.4401223425952452e+02,
       -5.7739158371735201e+01, -5.5071938546489875e+01,
       2.3412267991971751e+02, 1.0896706007924369e+02,
       -5.3757221801562757e+01, 3.0033182496318182e+02,
       1.0981931015988130e+02, -8.2474090196574451e+01,
       2.4673222510081393e+02, -7.3114850165927081e+01,
       -8.6719621112632851e+01, 2.3409346464950659e+02,
       7.1216843593141022e+01, -7.8163382971017697e+01,
       2.2240986055479209e+02, -5.1407677270735164e+01,
       -9.9501919712159520e+01, 2.6883353396441856e+02,
       -1.3848434814782354e+02, 3.9094398945545933e+01,
       2.4507040886959061e+02, 1.1151601041352653e+02,
       -9.1788845615317740e+00, 2.3802097377125125e+02,
       -1.1035766200012964e+02, 5.5222827276114730e+00,
       2.4701770931941985e+02, -8.6304573468152398e+01,
       -3.6083559888217458e+01, 2.0554022286665719e+02,
       1.1859810871166667e+02, 2.6719983002295997e+01,
       3.6885424220950898e+02, 1.0981619810955110e+02,
       -1.7738717481548071e+02, 4.1846993281482838e+02,
       -1.1952895949782774e+02, -2.7473034197011390e+01,
       2.8335950158289637e+02, -9.1190366218041675e+01,
       4.6077659461290366e+01, 2.2862069947216031e+02,
       -1.1209419108201378e+02, 9.7132742895369702e+01,
       2.4086819684532566e+02, -1.4433002252842419e+02,
       1.0696385813946058e+02, 2.3772505054078249e+02,
       6.7974917401599399e+01, 1.2029104569174066e+02,
       2.6003588141296279e+02, -1.1459999047901697e+01,
       1.2057258682575744e+02, 2.6438249327052966e+02,
       1.5060250612323810e+02, 1.1976758196715471e+02,
       2.6603590782042710e+02 ]
